examples/plot_callosal_tract_profile.py

```diff
@@ -50,7 +50,7 @@
 # Ensure segmentation logging information is included in this example's output
 root = logging.getLogger()
 root.setLevel(logging.ERROR)
-logging.getLogger('AFQ').setLevel(logging.INFO)
+logging.getLogger(__name__).setLevel(logging.INFO)
```

Comment:
unless really necessary, it'd be better not to redefine the logger level in a file like that

Reply:
This is an example, so its showing what the user can do. So I think it's good to show the getlogger and setlevel stuff. This file is not imported by any other file so it shouldn't affect anything else. That said I am not sure why there is this root logger thing.